sql_generators/mobile_kpi_support_metrics/templates/new_profiles.query.sql

```diff
@@ -0,0 +1,44 @@
+{{ header }}
+SELECT
+  first_seen_date,
+  normalized_channel,
+  app_name,
+  app_version,
+  country,
+  locale,
+  isp,
```

Comment:
i wonder if the cardinality of `isp` is too high? We originally had it in the [spec](https://docs.google.com/document/d/13aSkbugFFKH0e68uPmNzawtnvG5ShtRjCXoDhz2doH4/edit) as being only in the client table, but there may have been another ask to add it here, or maybe there is another reason to have it. 

No strong feelings from me if you think that the number of unique values is tolerable.

Reply:
I agree, we should remove it. I saw in the spec doc which is why included it. But I already chatted with Richard about removing device_model as this would also blow up the table.